src/main/java/org/ideoholic/curium/controller/Controller.java

```diff
@@ -70,8 +70,6 @@ private void processRequest(HttpServletRequest request,
      }else if("SubjectDetailsProcess".equalsIgnoreCase(process)){
          nextUrl= new SubjectDetailsAction(request, response).execute(action);
      }else if("printids".equalsIgnoreCase(process)){
```

Comment:
```suggestion
     }else if("FeesDetails".equalsIgnoreCase(process)){
```
Don't delete the ```FeesDetails``` if-condition, remove the ```printids``` if-condition